Doppler.ContactPolicies.Business.Logic/Services/IUserFeaturesService.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Business.Logic.Services
+{
+    public interface IUserFeaturesService
+    {
+        Task<bool> GetUserContactPoliciesFeature(string accountName);
```

Comment:
Just a detail to continue with naming conventions 
```suggestion
        Task<bool> GetUserContactPoliciesFeatureAsync(string accountName);
```